__init__.py

```diff
@@ -288,23 +327,19 @@ def __init__(self):
         self.CODES['50d', '50n'] = 7                # windy/misty
 
         # Use Mycroft proxy if no private key provided
-        self.settings["api_key"] = None
-        self.settings["use_proxy"] = True
+        self.settings.setdefault("api_key", None)
+        self.settings.setdefault("use_proxy", True)
 
     def initialize(self):
-        # TODO: Remove lat,lon parameters from the OWMApi()
-        #       methods and implement _at_coords() versions
-        #       instead to make the interfaces compatible
-        #       again.
```

Comment:
I interpreted this as splitting the location methods to make the wrapper interfaces consistent with the OWM API. Though I find the current approach cleaner. A single call for location based queries, and lat, lon would always be considered to be more accurate than a general place name so are prioritised.
Just raising it in case anyone wants to argue the other side before we remove the # TODO

Reply:
I also think having just one call would be more pythonic